Formula/mongo-cxx-driver.rb

```diff
@@ -0,0 +1,35 @@
+class MongoCxxDriver < Formula
+  desc "C++ driver for MongoDB"
+  homepage "https://github.com/mongodb/mongo-cxx-driver"
+  url "https://github.com/mongodb/mongo-cxx-driver/archive/r3.0.3.tar.gz"
+  sha256 "297957788ed6f8d32ff05cf1a92f022aa4e3cd4398b0c2a0d4a2434363fbde6d"
+  head "https://github.com/mongodb/mongo-cxx-driver.git"
+
+  bottle do
+    sha256 "4907896f59be2159c14c5d113e7bd5aed8bd5e53b9b8d9f8bfc033179e3f15a4" => :sierra
+    sha256 "52a5b6ac0764be74eea200b9e3012982622762cc2f45e31896819750b1fa4a8d" => :el_capitan
+    sha256 "b86a75b625b28a5378c23678a0d02d42b0bfeb01f64d16a7eee111bfa97284b0" => :yosemite
+  end
+
+  needs :cxx11
+
+  depends_on "cmake" => :build
+  depends_on "mongo-c-driver"
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make"
+    system "make", "install"
+    pkgshare.install "examples"
+  end
+
+  test do
+    system ENV.cc, "-o", "test", pkgshare/"examples/mongocxx/connect.cpp",
+      "-I#{include}/mongocxx/v_noabi",
+      "-I#{include}/bsoncxx/v_noabi",
+      "-I#{Formula["mongo-c-driver"].include}/libmongoc-1.0",
+      "-I#{Formula["libbson"].include}/libbson-1.0",
```

Comment:
Should this be `Formula[mongo-c-driver]` now that `libbson` is no longer an independent formula?